### ISSUE TITLE
Voice calls don't work after upgrade from 4.6.5 to 4.7

### ISSUE THREAD

**Issue by zaps166** [telegramdesktop/tdesktop#26066]
### Steps to reproduce

1. Start Telegram 4.7 on Linux
2. Call to person on Telegram 4.7 on Windows or Android

### Expected behaviour

Voice, camera, screen sharing are working correctly.

### Actual behaviour

- call window is visible and timer is running
- no audio (audio not open in PipeWire/PulseAudio, application not visible in `pavucontrol`)
- camera/screen sharing is not streamed to calle

---

- still works: Windows<->Android or Linux<->Linux
- no longer works: Linux<->Windows or Linux<->Android.

Not related to tdesktop, but might help: Android<->Android call usually has poor audio quality, I have to create group with 2 people and call on group to workaround this issue.

### Operating system

Manjaro Linux, Xfce4

### Version of Telegram Desktop

4.7

### Installation source

Static binary from official website

**Comment by VladimirMrzv** [comment]
it is broken from 4.6.10 beta.

Tested and not works:
linux <-> linux  (all with beta)
linux <-> android
linux <-> ios 
linux <-> mac

works:
mac <-> android
linux <-> linux both in one network
linux 4.6.5 <-> linux 4.6.12

**Comment by Ruok2bu** [comment]
I downgraded telegram just now to fix mine. Here's how to do that (i use flatpak version so first uninstall other version if you use it):

1. As root user, in terminal type: flatpak remote-info --log flathub org.telegram.desktop
2. To downgrade to previous version, type: latpak update --commit=76dad107dc7167c38d841c8f5d3634614154506f811a4001e4ecb0938e288a58 org.telegram.desktop

I tested after doing that and calls work again! Just remember not to upgrade until its fixed!
